ldap/servers/slapd/daemon.c

```diff
@@ -2083,7 +2082,7 @@ unfurl_banners(Connection_Table *ct, daemon_ports_t *ports, PRFileDesc **n_tcps,
     char addrbuf[256];
     int isfirsttime = 1;
 
-    if (ct->size <= slapdFrontendConfig->reservedescriptors) {
+    if (ct->size > (slapdFrontendConfig->maxdescriptors - slapdFrontendConfig->reservedescriptors)) {
```

Comment:
This code is hit on startup, so we can only use the static values of descriptor values